src/desktop_release.js

```diff
@@ -212,6 +212,10 @@ class DesktopReleaseBuilder {
         // so load it here
         const cfg = JSON.parse(await fsProm.readFile(path.join(repoDir, 'package.json'))).build;
 
+        // Electron crashes on debian if there's a space in the path.
+        // https://github.com/vector-im/riot-web/issues/13171
+        if (type === 'linux') cfg.productName = 'Element';
```

Comment:
Won't this clobber Nightly and cause the packages to clash?

Reply:
nightly should still be 'element nightly' (that's done by the other file)

Reply:
Rather than setting to a fixed value, could we do our own space to hyphen conversion to avoid a collision perhaps...?

Reply:
ah yep, if we're happy for it to be hyphenated

Reply:
I meant, it's ugly, but also you don't really see the directory name on Linux unless you go hunting for it.

Reply:
(You still use `element-desktop` to launch it, so the directory is mostly a container the user doesn't need to care much about.)

Reply:
Ah but I guess it affects desktop shortcuts...

Reply:
It will also be the name it appears as generally though

Reply:
Okay, I guess this is fine for release then. For nightly, could you prepare the same with `Element-Nightly` or something? I think it's acceptable for that to be uglier that the official release.

Reply:
Did you meant just using 'element' or the hyphenating? It's currently doing the hyphenating.

Reply:
Sorry I keep changing my mind... 😭 I think just "Element" here as you already have it probably better for release, since as you say, it appears in app launchers and things.